generators/go/internal/writer/workflows/ci.yml

```diff
@@ -23,6 +23,16 @@ jobs:
       - name: Set up go
         uses: actions/setup-go@v4
 
+      - name: Log in to Docker Hub
+        if: ${{ env.DOCKER_USERNAME != '' && env.DOCKER_PASSWORD != '' }}
+        uses: docker/login-action@v3
+        with:
+          username: ${{ env.DOCKER_USERNAME }}
+          password: ${{ env.DOCKER_PASSWORD }}
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
I'd like to avoid touching the user-facing ci.yml files for this one. Too big of a change.

Can we address the change with how we run seed instead?